bytes/bytes/models.py

```diff
@@ -50,6 +50,9 @@ class Job(BaseModel):
     started_at: AwareDatetime
     ended_at: AwareDatetime
 
+    def __hash__(self) -> int:
+        return hash(self.id)
```

Comment:
Since UUIDs are designed to be unique, `self.id` can serve as a hash or unique identifier

```suggestion
        return self.id.int
```